app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,31 @@
+class FriendshipsController < ApplicationController
+  def create
+    send_request
+    flash[:notice] = 'Request sent successfully'
+    redirect_to users_path
+  end
+
+  def reject
+    @request = Friendship.get_request(current_user, params[:sender_id])[0]
+    @request.destroy
+    redirect_to received_requests_path
+  end
+
+  def accept
+    @request = Friendship.get_request(current_user, params[:sender_id])
+    @request.update(status: 'accepted')
+    Friendship.create(sender_id: current_user.id,
+                      receiver_id: params[:sender_id], status: 'accepted')
```

Comment:
As this concern updating the model, please move it into `Friendship` model
```rb
def confirm_friend
  self.update_attributes(status: 'accepted')
  Friendship.create!(sender_id: self.receiver_id,
                      receiver_id: self.sender_id
                  status: 'accepted')
end
```
And call it like this in your controller
```rb
@request.confirm_friend
```